client/state/selectors/get-site-feed-url.ts

```diff
@@ -0,0 +1,15 @@
+import 'calypso/state/data-layer/wpcom/sites/scan';
+import getSiteUrl from './get-site-url';
+import type { AppState } from 'calypso/types';
+
+/**
+ * Returns the current Jetpack Scan request status for a given site.
+ * Returns undefined if the site is unknown, or if no information is available.
+ */
+export default function getSiteFeedUrl( state: AppState, siteId: number ): string | null {
+	const siteUrl = getSiteUrl( state, siteId );
+	if ( ! siteUrl ) {
+		return null;
+	}
+	return `${ siteUrl }/feed`;
+}
```

Comment:
I am not much familiar with Jetpack Scan. What is the main purpose of this function? I mean it is clear to me what it does in its core but I don't see it used elsewhere in the code. 🤔

Reply:
Ops. I created this but then decided not to use it, I was meant to delete the file. Thanks for noticing this!

Reply:
My pleasure! :)